gtest/src/program.rs

```diff
@@ -353,32 +353,40 @@ impl<'a> Program<'a> {
         )
     }
 
-    pub fn send<ID: Into<ProgramIdWrapper>, C: Codec>(&self, from: ID, payload: C) -> RunResult {
-        self.send_with_value(from, payload, 0)
+    pub fn send<ID: Into<ProgramIdWrapper>, C: Codec>(
+        &self,
+        from: ID,
+        payload: C,
+        delay: Option<u32>,
```

Comment:
that's exactly programs interface. is we discussed before, users have no ability to send delayed messages. it's programs specific feature atm

Reply:
Furthermore, delay also shouldn't be optional anywhere in code: in case of absence of delay (when you want to send something instantly) delay just equals 0